instances/technique/epidermalElectrophysiologyTechnique.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/epidermalElectrophysiologyTechnique",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "The term 'epidermal electrophysiology technique' refers to any electrophysiology technique where electrodes are placed onto the skin of an organism to non-invasively (or minimally invasively) measure electrical functions of it's body.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "epidermal electrophysiology technique",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "cutaneous electrophysiology",
+    "cutaneous electrophysiology technique",
```

Comment:
Is this correct? These seem more like related terms and not exact synonyms. "cutaneous" is only "related to the skin", "epidermal" is more specific.

Reply:
not sure. This article seems to use it as a synonym: https://www.nature.com/articles/s41551-019-0365-8 What do you think?

Reply:
Yeah, I also came across that one but they only use cutaneous once in the first sentence. I got the impression that they were speaking of cutaneous techniques in general. My interpretation was that epidermal techniques are a subclass of cutaneous techniques, but I could be wrong 😉

Reply:
don't know either. let's delete them (we can always add them again)